scripts/run-module-bundlers-smoketests.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+const shell = require("shelljs");
+
+let result = 0;
+
+console.log(`
+Test webextension-polyfill bundled with webpack
+===============================================`);
+
+shell.exec("webpack --mode production --entry ./test/fixtures/bundle-entrypoint.js --output /tmp/webpack-bundle.js");
+process.env.TEST_BUNDLED_POLYFILL = "/tmp/webpack-bundle.js";
```

Comment:
Nit, in my opinion setting the process.env.TEST_BUNDLED_POLYFILL and then the two shell.exec calls may be a bit more readable, I mean something like:

```
process.env.TEST_BUNDLED_POLYFILL = "...";
shelljs.exec("webpack ...");
shelljs.exec("npm run test"):
```